### PR TITLE
Minor rephrase/format in the installation instructions

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,10 +88,9 @@ pip install regional-mom6
 
 The above installs the version of `regional-mom6` (plus any required dependencies) that corresponds to the latest tagged release of the package.
 
-#### "*I'd like to be on the cutting edge of the development*?"
+#### "*I want to live on the edge! I want the most latest developments*"
```
Comment (angus-g) [inline README.md:91]:
```suggestion
#### "*I want to live on the edge! I want the latest developments*"
```

File: docs/installation.md [hunk 2]
```diff
@@ -37,17 +36,16 @@ pip install regional-mom6
 
 The above installs the version of `regional-mom6` (plus any required dependencies) that corresponds to the latest tagged release of the package.
 
-#### "*I'd like to be on the cutting edge of the development*?"
+## "*I want to live on the edge! I want the most latest developments*"
```
Comment (angus-g) [inline docs/installation.md:39]:
```suggestion
## "*I want to live on the edge! I want the latest developments*"
```

File: docs/installation.md [hunk 3]
```diff
@@ -37,17 +36,16 @@ pip install regional-mom6
...
-First install `esmpy` as described above and then:
+To install `regional-mom6` directly via GitHub using `pip`, first install `esmpy` as described above. Then:
 
 ```bash
 pip install git+https://github.com/COSIMA/regional-mom6.git
 ```
 
 to get the version that corresponds to the latest commit in GitHub.
-Or, install the version that corresponds to a particular git commit using
+Alternatively, install the version that corresponds to a particular git commit using
```
Comment (angus-g) [inline docs/installation.md:48]: This doesn't get to be in the readme? :wink:

File: README.md [hunk 4]
```diff
@@ -52,10 +52,11 @@ Check out the [documentation](https://regional-mom6.readthedocs.io/en/latest/) a
...
 ## Installation
 
-#### Easy, clean, one liner
+We encourage creating a new or using an existing conda environment.
 
-The easiest way is to install `regional-mom6` via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
-We encourage creating a new or using an existing conda environment and then simply
+#### Easy, clean, one liner via conda
+
+The easiest way is to install `regional-mom6` is via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
```
Comment (angus-g) [inline README.md:59]:
```suggestion
The easiest way to install `regional-mom6` is via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
```

File: docs/installation.md [hunk 5]
```diff
@@ -1,21 +1,20 @@
...
+# Installation
 
-#### Easy, clean, one liner
+We encourage creating a new or using an existing conda environment.
 
-The easiest way is to install `regional-mom6` via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
-We encourage creating a new or using an existing conda environment and then simply
+## Easy, clean, one liner via conda
+
+The easiest way is to install `regional-mom6` is via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
```
Comment (angus-g) [inline docs/installation.md:7]:
```suggestion
The easiest way to install `regional-mom6` is via [`conda`](https://anaconda.org/conda-forge/regional-mom6).
```

**Review by angus-g — COMMENTED** [review]
Few little tweaks, but otherwise fine!

File: docs/installation.md (hunk 3 above)
Comment (navidcy) [reply docs/installation.md:48]: nice catch! dealt with

**Review by angus-g — APPROVED** [review]

**Pull request merged**